mlflow/protos/service.proto

```diff
@@ -469,6 +469,18 @@ service MlflowService {
       rpc_doc_title: "Search Traces",
     };
   }
+
+  rpc deleteTraces(DeleteTraces) returns (DeleteTraces.Response) {
+    option (rpc) = {
+      endpoints: [{
+        method: "POST",
+        path: "/mlflow/databricks/traces/delete-traces"
```

Comment:
Given it's not a databricks-only API, we can remove the "databricks/" here. 
```suggestion
        path: "/mlflow/traces/delete-traces"
```

Reply:
@liangz1 Thanks for the catch! Filed https://github.com/mlflow/mlflow/pull/11575